3 - Assignment/240326 - Rust Basic + Roguelike Game, Week 1/nested-arrays/src/lib.rs

```diff
@@ -1,5 +1,15 @@
 pub fn transpose(matrix: [[i32; 3]; 3]) -> [[i32; 3]; 3] {
-    todo!("Implement this")
+    let mut ret = [
+        [0,0,0],
+        [0,0,0],
+        [0,0,0],
+    ];
+    for i in 0..matrix.len(){
+        for j in 0..matrix[i].len(){
+            ret[j][i] = matrix[i][j];
+        }
+    }
+    return ret;
```

Comment:
1. `return`을 생략할 수 있습니다.
2. `cargo fmt`를 통해 코드 스타일을 맞춰주세요.

```suggestion
    for i in 0..3 {
        for j in 0..3 {
            ret[i][j] = matrix[j][i];
        }
    }

    ret
```